requirements.txt

```diff
@@ -3,3 +3,5 @@ networkx>=2.5
 PyYAML==6.0
 requests>=2.22
 tamr_unify_client>=1.4.0
+boto3>=1.21.21
```

Comment:
```suggestion
```
Please remove this line. This package is only needed as part of the `optional_requirements.txt` file.